### DISCUSSION TITLE
VIA point - location method

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20840] [General]
I made a route:

Start
1 - VIA point
2 - VIA point
Destination

I got to the first point (1 - VIA point) and found parking far away 200m.
He got back in the car and continued on to the next stop (2 - VIA point). App takes me back to"1 - VIA point".

1. The application should detect that I have continued further towards the point (2 - VIA point). 
2. Good navigations, VIA point treated as an area and not a point. And they never return to the previous VIA point. Only the Polish navigation 'Automap' has an option when selecting VIA point to assign a location method, such as an area or a point.

**Comment by sjvudp** [comment]
Or the app should provide forward/back buttons that select the next or previous via point (I think airplane GPS have such a function).
TomTom had a function like "skip next target" to continue with the second-to-next via (re-calculating the route).

**Reply by ZeljkoBG** [reply]
That. 'Skip...' have all navigations.
